airflow/www/static/js/ti_log.js

```diff
@@ -153,4 +171,10 @@ $(document).ready(() => {
     const autoTailing = i === TOTAL_ATTEMPTS;
     autoTailingLog(i, null, autoTailing);
   }
+
+  setDownloadUrl();
+  $('#ti_log_try_index_list a').click(function () {
+    const tryNumber = $(this).attr('href').replace(/\D/g, '');
+    setDownloadUrl(tryNumber);
```

Comment:
Regex doesn’t feel like a good idea here. Maybe put this value in a data attribute instead?

Reply:
Great idea, I'll add that in